### PR TITLE
Fmincon update

### PR DESCRIPTION
**Pull request by pgkirsch**
PR to get latest version of fmincon code merged

### CODE REVIEW

**Comment by whoburg** [comment]
test models please

File: gpkit/tools/fmincon.py [hunk 1]
```diff
@@ -126,6 +172,10 @@ def generate_mfiles(model, algorithm='interior-point', guesstype='ones',
                  "end")
 
     # String for main.m
+    if logspace:
+        fval = "exp(fval)"
+    else:
+        fval = "fval"
```
Comment (whoburg) [inline gpkit/tools/fmincon.py]: `fval = "exp(fval)" if logspace else "fval"`

**Comment by whoburg** [comment]
test models please

**Comment by whoburg** [comment]
test models please

**Comment by whoburg** [comment]
test models please

**Comment by bqpd** [comment]
test models please

**Comment by bqpd** [comment]
test models please

**Comment by whoburg** [comment]
made the minor change I suggested in my review. This has my approval. Leaving to @bqpd to merge.

**Comment by bqpd** [comment]
Awesome! Mergin'

**Pull request merged**